### PR TITLE
DWR-489: Info popovers in grid headers

### PR DESCRIPTION
**Pull request by jtreuting**
- Implemented for 3 columns, but commented out for 1 of them for now since it messes the wrapping and a demo is coming up
- trigger of focus doesn't work on a span or an anchor tag so implemented show on click and hide on mouseleave.  don't love it so thoughts appreciated on this

### CODE REVIEW

File: webapp/src/main/webapp/src/app/groups/groups.module.ts [hunk 1]
```diff
@@ -32,7 +33,8 @@ import { SelectAssessmentsComponent } from "./results/select-assessments/select-
     BrowserAnimationsModule,
     FormsModule,
     DataTableModule,
-    SharedModule
+    SharedModule,
+    PopoverModule.forRoot()
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/groups/groups.module.ts]: This is where I'm a little fuzzy on .forRoot() If we use .forRoot() should it be in the app module?

File: webapp/src/main/webapp/src/app/groups/results/assessment/assessment-results.component.html [hunk 2]
```diff
@@ -87,19 +87,44 @@ <h3 class="gray-darkest mb-md">
             </ng-template>
           </p-column>
           <p-column field="session" header="{{'labels.groups.results.assessment.exams.cols.session' | translate}}" [sortable]="true"></p-column>
-          <p-column field="enrolledGrade" header="{{'labels.groups.results.assessment.exams.cols.grade' | translate}}" [sortable]="true" ></p-column>
-          <p-column field="administrativeCondition" header="{{'labels.groups.results.assessment.exams.cols.status' | translate}}" [sortable]="true" >
+          <p-column field="enrolledGrade" header="{{'labels.groups.results.assessment.exams.cols.grade' | translate}}" [sortable]="true" [style]="{'text-align':'center'}" ></p-column>
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/groups/results/assessment/assessment-results.component.html]: Recommend no direct-injected styles (or [style] which creates a data-binding to a constant.)

File: webapp/src/main/webapp/src/app/groups/results/assessment/assessment-results.component.ts [hunk 3]
```diff
@@ -121,6 +121,10 @@ export class AssessmentResultsComponent {
       (this.isIab ? "iab" : "ica") + ".performance";
   }
 
+  get performanceLevelHeaderInfo() {
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/groups/results/assessment/assessment-results.component.ts]: Is this being used?  It looked like you had to hard-code the status one

File: webapp/src/main/webapp/src/app/groups/results/assessment/assessment-results.component.html (hunk 2 above)
Comment (jtreuting) [inline webapp/src/main/webapp/src/app/groups/results/assessment/assessment-results.component.html]: good call, will change

File: webapp/src/main/webapp/src/app/groups/results/assessment/assessment-results.component.ts (hunk 3 above)
Comment (jtreuting) [inline webapp/src/main/webapp/src/app/groups/results/assessment/assessment-results.component.ts]: yeah, it's used in the commented out one for the performance column.  It is a dynamic column header and info based on whether it is IAB or ICA.  IAB calls it Reporting Category and ICA calls it Achievement Level but we back it with the same data.

**Pull request merged**